lib/gosu/patches.rb

```diff
@@ -57,6 +57,31 @@ def tick
   end
 end
 
+class Gosu::Image
+  def ==(img)
+    self.to_blob == img.to_blob
+  end
+
+  # Checks if two images are similar on a really basic level (check the difference of each channel)
+  def similar_to?(img, treshold=0.90)
```

Comment:
Typo: *`threshold`

I think `similar?(img)`  would be more in line with Ruby's `equal?`. And I don't think that this method is useful outside of tests, so it can go into the `test_helper.rb` file for now.